javascript/react/intro-react/why-use-react.md

```diff
@@ -18,24 +18,22 @@ aspects:
   - introduction
 
 ---
-# Why use **React**?
+# Why use React?
 
 ---
 ## Content
 
-**React** is considered a great choice for building the **UI** of web applications - be it simple *SPAs*[1] or
-large projects with numerous authors due to the **self-containing** nature of its *components*.
+**React** is considered a great choice for building the **UI** of web applications - be it simple *SPAs*[1] or large projects with numerous authors due to the **self-containing** nature of its *components*.
 
 In addition to the previously stated reasons developers are eager to adopt this library, **React** comes with many more interesting features such as:
 
 - great *performance* as a result of the **Virtual DOM** diff algorithm
-- **React**'s syntax is simple. Unlike other popular `JS` frameworks with specific syntax, **React** makes use of pure `JS`
+- **React**'s syntax is simple. Unlike other popular JS frameworks with specific syntax, **React** makes use of pure JS
```

Comment:
We probably shouldn't compare it to other frameworks, particularly in this case where "simplicity" is a subjective trait.

We should instead talk about how the syntax is ["declarative"](https://stackoverflow.com/questions/129628/what-is-declarative-programming)

Reply:
changed bullet point to:

- **React**'s syntax is declarative[3]

[3:Declarative Syntax]
This means that you write your code such that it describes what you want to do, not how you want to do it. Figuring out how is left to the compiler.